### ISSUE TITLE
Download Icon in top right corner where I can see current downloads.

### ISSUE THREAD

**Issue by clicky6** [brave/brave-browser#18468]
**Improvements for download bar**
1. Because I’ve already read some posts here about the thickness and the missing ability to hide it (the general behavior of this bar partly annoys me myself), maybe it’s time to rethink it.

**Current state:**
Big & clunky
You have to close it manually
You can’t deactivate it
Doesn’t have any useful purpose except showing that something is downloading

**Ideas of improvement:**
a) Make it smaller or enable the option to hide it completely or
b) (as in better)
Wouldn’t it be a good idea if this download bar could serve as a kind of tray to which you can drag all the objects you want to download while browsing? Related to the particular tab or window you are working in.

You could then download all the things in one rush. Or staggered. That would first **be intuitive** and secondly, the current big size of this bar would also **have a purpose**.

Source of idea:
A little tool for macOS called [Dropover](https://dropoverapp.com/).

**Adopt new download UI from MS Edge with mini downloads window**
2. I would like to request a modification to the Downloads UI which mimics that which is seen in MS Edge.
When initiating a download, instead of putting the downloads bar at the bottom of the screen they have added a temporary download icon in the top-right corner which allows for a small pop-up UI on monitor, pause and open downloads as well as clear the download list directly without having to go to the download screen.

This is a much neater UI than the default Chrome experience, faster to interact with and doesn’t necessitate me moving between different tabs repeatedly to manage my download list.

![image](https://user-images.githubusercontent.com/89839714/135409908-9995cd82-0c25-4719-a30c-13dbb351c92f.png)

**Download Icon in top right corner where I can see current downloads**
3. It would be so much easier to access downloads window on Brave.
![image](https://user-images.githubusercontent.com/89839714/135410389-609eab26-2269-48c2-9fbe-7ffb5be1d6f9.png)

These are posts from different users from Brave Community Feature-Requests, I just combined them and posted them on GitHub.
https://community.brave.com/t/improvements-for-download-bar/245592 by sherold.
https://community.brave.com/t/adopt-new-download-ui-from-ms-edge-with-mini-downloads-window/247452 By Desmo46.
https://community.brave.com/t/download-icon-in-top-right-corner-where-i-can-see-current-downloads/286182 By TechBoy.

**Comment by Nihal247** [comment]
First of all why to post here, when the posts are already submitted at the Brave Community, just to increase your contributions? C'mon @clicky6, I've seen you posting useless feature requests, making Brave's _brave_  developers work more, when the currents are already awesome. No need for all of these, we're not here for creating a Microsoft Edge fork.

**Comment by clicky6** [comment]
@Nihal247 
> First of all why to post here, when the posts are already submitted at the Brave Community, just to increase your contributions? C'mon @clicky6, I've seen you posting useless feature requests, making Brave's _brave_ developers work more, when the currents are already awesome. No need for all of these, we're not here for creating a Microsoft Edge fork.

Extremely Sorry,
I posted many feature requests on Brave Community but they didn't seem to come into development and there was no response from developers and so I started posting them on GitHub.
I just posted the things which Microsoft Edge seems to be better at ,Microsoft Edge's visuals are somewhat better. The browser looks and feels very good, and I am not saying that Brave should copy all these things, It is not necessary but It would improve the user experience and Brave could at least take the good components of other browsers and design it with their own style and then put it in the browser. I was just pointing out the things which can be improved. If they feel that these features are necessary they could add it else leave it...
